aqt/addons.py

```diff
@@ -302,6 +303,13 @@ def redrawAddons(self):
         if self.addons:
             self.form.addonList.setCurrentRow(0)
 
+    def _onAddonItemSelected(self, row_int):
+        try:
+            addon = self.addons[row_int][1]
+        except:
```

Comment:
You should never have a bare "except", you should specify which exception to catch.